### ISSUE TITLE
04_model

### ISSUE THREAD

**Issue by Benjamintdk** [Benjamintdk/DSAI-Project#7]
1) Can we use transformer architecture with pretrained encoders and decoders already? Reduces headache of deciding how many decoders to use. --> Stick with untrained decoders, use [SEP] token as starting token.  

2) How to invert the word embedding layer to use as final output layer instead of simple linear layer? If cannot, do we use vanilla embedding or linear layer? --> Just stick with linear layer for simplicity, followed by softmax. 

3) How to generate final output sequence? Do we need a for loop? --> Yes we do based on source code. Set to max length. This should be set when creating the datasets too.
